<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bmi</class>
 <widget class="QDialog" name="bmi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="BMI" extracomment="BMI">BMI</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="0" rowspan="4">
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="1" alignment="Qt::AlignHCenter">
        <widget class="QLineEdit" name="lineEdit_plec">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="8" column="0" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Oblicz BMI:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_wzrost">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Wpisz wzrost [cm]:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" alignment="Qt::AlignHCenter">
        <widget class="QLineEdit" name="lineEdit_bmi">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_waga">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Wpisz wagę [kg]:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" alignment="Qt::AlignHCenter">
        <widget class="QLineEdit" name="lineEdit_waga">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="5" column="0" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_plec">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Wpisz płeć [k/m]:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" alignment="Qt::AlignHCenter">
        <widget class="QLineEdit" name="lineEdit_bmi_2">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QPushButton" name="pushButton_wroc">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Wróć</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_bmi_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Twoje BMi jest:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" alignment="Qt::AlignHCenter">
        <widget class="QLineEdit" name="lineEdit_wzrost">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="6" column="0" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_bmi">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Twoje BMI wynosi:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>BMI</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>75</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="plainText">
        <string>Wskaźnik masy ciała, także wskaźnik Queteleta II – współczynnik powstały przez podzielenie masy ciała podanej w kilogramach przez kwadrat wysokości podanej w metrach. Klasyfikacja wskaźnika BMI została opracowana wyłącznie dla dorosłych i nie może być stosowana u dzieci</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
